res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
We need a solution that resolves Leap as well. Otherwise that's the entire stable desktop platform of openSUSE left out. And in resolving it for Leap, that will resolve it for Tumbleweed as well.